Queue Implementation using Stack

```diff
@@ -0,0 +1,52 @@
+class MyQueue:
+
+    def __init__(self):
+        """
+        Initialize your data structure here.
+        """
+        self.stack1= []
+        self.stack2= []
+        
+
+    def push(self, x: int) -> None:
+        """
+        Push element x to the back of queue.
+        """
+        self.stack1.append(x)
+        
+        
+
+    def pop(self) -> int:
+        """
+        Removes the element from in front of queue and returns that element.
+        """
+        if self.stack2==None:
```

Comment:
None is a data type used for defining objects. It is not zero or false. We can assign None to a variable. Use [] or 0 for the code to run. 
i.e.  if self.stack2==[]:
            while self.stack1 !=[]:
                self.stack2.append(self.stack1.pop())
        return self.stack2.pop()